wal/encoder.go

```diff
@@ -36,7 +36,7 @@ type encoder struct {
 
 func newEncoder(w io.Writer, prevCrc uint32) *encoder {
 	return &encoder{
-		bw:  bufio.NewWriter(w),
+		bw:  ioutil.NewPageWriter(w, 4096),
```

Comment:
Also doc that this is buffering same as bufio https://github.com/golang/go/blob/master/src/bufio/bufio.go#L18?